scalameta/quasiquotes/shared/src/main/scala/scala/meta/internal/quasiquotes/ReificationMacros.scala

```diff
@@ -122,10 +126,7 @@ class ReificationMacros(val c: Context) extends AstReflection with AdtLiftables
         }
         curr + 1
       }
-      val result = {
-        if (reflectInput.file.file != null) Input.File(reflectInput.file.file)
-        else Input.String(new String(reflectInput.content)) // NOTE: can happen in REPL or in custom Global
-      }
+      val result = Input.String(new String(reflectInput.content))
```

Comment:
I'm afraid this may degrade error reporting for invalid quasiquotes. Is it possible to restore the initial version?